### ISSUE TITLE
AttributeError: 'ServerPrx' object has no attribute 'updateCertificate'

### ISSUE THREAD

**Issue by Sxderp** [Astraalivankila/Update-Mumble-Server-Certificates#1]
Tried running the script and this is the error I receive. Using Murmur 1.2.10 on FreeBSD (through ports).
I checked the APIs that I could find online, and I didn't see any reference to an updateCertificate() function
on the Server object (for any version).

I'm a bit confused.

**Comment by oplehtinen** [comment]
Did you install and configure mice? updateCertificate() is an ICE command.

**Comment by Sxderp** [comment]
Yes, however, after looking back at the mice.py code the issue may be with the location of my Murmur.ice. 

Currently sitting in a car driving north (I'm not driving). I'll ssh to my server and check when I get somewhere with WiFi.

**Comment by oplehtinen** [comment]
Yeah. The script should be in the same directory with mice.

**Comment by oplehtinen** [comment]
Sorry I misread your earlier post. Did you figure it out?

**Comment by Sxderp** [comment]
Yes, no, yes. (lol).

Firstly, yes, I had the Murmur.ice in a different location. I fixed the path.

No, this did not resolve the underlining issue.

Yes, I figured out what the problem is. FreeBSD ports has [1.2.10](https://github.com/mumble-voip/mumble/releases/tag/1.2.10) as the latest version (to be updated eventually). However, 'updateCertificate' was added fairly recently [here](https://github.com/mumble-voip/mumble/pull/2221). Thanks for your help anyway.
